tests/scenarios/example.py

```diff
@@ -6,7 +6,7 @@
 # Basic information about ceph and its configuration
 ceph = {
     'releases': ['jewel', 'infernalis'],
-    'cluster_name': 'ceph'
+    'cluster_name': 'test'
```

Comment:
@leseb nope, that entire example could actually be removed. In https://github.com/ceph/ceph-ansible/pull/1150 we removed the need for a ``scenario.py`` file at all.